TombLib/TombLib/Catalogs/TEN Node Catalogs/Timespan Actions.lua

```diff
@@ -161,11 +161,29 @@ end
 -- !Description "Gradually change colour of a moveable over specified timespan."
 -- !Arguments "NewLine, Moveables, 70" "Color, 15, Moveable colour"
 -- !Arguments "Numerical, [ 0.1 | 65535 | 2 | 0.1 | 1 ], {1}, 15, Time (in seconds)"
+-- !Arguments "NewLine, Numerical, [ 0 | 255 | 2 | 1 | 1 ], {255}, 15, Alpha"
 
-LevelFuncs.Engine.Node.ChangeMoveableColourOverTimespan = function(moveableName, newColour, time)
+LevelFuncs.Engine.Node.ChangeMoveableColourOverTimespan = function(moveableName, newColour, time, alpha)
+	newColour.a = alpha
 	do LevelFuncs.Engine.Node.ConstructTimedData(moveableName, false, 3, newColour, time, true) end
 end
 
+-- !Name "Change alpha of a moveable"
```

Comment:
I suggest to use word "transparency" instead of "alpha". For builders without experience with graphics terminology it may sound as something unknown.